### DISCUSSION TITLE
QField - Can't access local storage on iOS phone

### DISCUSSION THREAD

**Discussion by duboisAM** [opengisch/QField#3755] [Q&A]
I usually use Android products with QField but right now I need to evaluate all portable solutions, which includes iOS. However, just when I come to the data storage access in order to load a new project, I hit a wall. I have been able to successfully load projects from QFieldCloud, but I can't seem to be able to add anything else, unless it has been previously loaded in the specific QField folder (created when the app is installed) or the already selected "favorite folders". Am I missing something here?

I have browsed on QField Ecosystem Documentation website in order to find an answer (https://docs.qfield.org/get-started/storage/), but there is only info for Android OS. Because I don't find it very user-friendly if there are no ways to access other storage solutions (OneDrive, GoogleDrive, DropBox) or even just another local storage folder on my device.

In French we say, 1 image makes up for a 1000 words. I hope this example will make my question clearer: on left iOS data access page (no options to further search), on right Android data access page (with big green plus to get further options)
![image](https://user-images.githubusercontent.com/64656876/208479174-f5ae7a80-7f30-4534-981d-52cde820ae68.png)

**Comment by nirvn** [comment]
@duboisAM , import functionalities on iOS haven't been implemented yet. You can however drag and drop files and projects folders into the QField app directory via iTunes when you plug in your phone using a USB cable.

On iOS, import actions are most likely to be implemented quickly if someone steps up to sponsor the effort.

**Reply by yaoyuandejuli** [reply]
Hello , 

What about if some picture where not pushed through correctly and you need to manually dump them in  the DCIM folder so that they can and up in the cloud .
How would one retrieve picture on a Iphone or alternatively is there a way to push them back again ?

**Comment by duboisAM** [comment]
@nirvn It turns out that if 
1) I zip my project (I used QPackage plugin to copy all related files to one folder)
2) copy it directly into the Qfield folder (that was more intricate than I expected since I didn't want to use iTunes. I had to access my OneDrive via the Files app to make the copy directly into the QField files directory), 
3) then unzip the folder,

I can FINALLY see my project in the QField files directory from QField app homepage.
Victory!

**Reply by MrKappa** [reply]
Thanks. I almost did the same procedure except that I've used Google Drive and saved the file locally (zip archive). Then I moved the file with the Files app to the QField folder.
